test/clt-tests/sharding/replication/replication-test.rec

```diff
@@ -0,0 +1,302 @@
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export INSTANCE=3
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export INSTANCE=4
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export INSTANCE=5
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+export CLUSTER_NAME=replication
+––– output –––
+––– block: ../base/flexible/create-cluster –––
+––– input –––
+export CLUSTER_NAME=replication
+––– output –––
+––– block: ../base/flexible/join-cluster-on-all-nodes –––
+––– input –––
+mysql -h0 -P1306 -e "CREATE TABLE testrt (title text, content text, gid uint);"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "INSERT INTO testrt VALUES(0,'List of HP business laptops','Elitebook Probook',1);"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "INSERT INTO testrt VALUES(6775836565403860994,'List of Dell business laptops','Latitude Precision Vostro',2);"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "ALTER CLUSTER replication ADD testrt;"; echo $?
+––– output –––
+0
+––– input –––
+mysql -h0 -P1306 -e "SHOW STATUS LIKE 'cluster_replication_indexes';"
+––– output –––
++-----------------------------+--------+
+| Counter                     | Value  |
++-----------------------------+--------+
+| cluster_replication_indexes | testrt |
++-----------------------------+--------+
+––– input –––
+mysql -h0 -P1306 -e "SELECT * FROM testrt\G;"
+––– output –––
+*************************** 1. row ***************************
+id: 6775836565403860994
+title: List of Dell business laptops
+content: Latitude Precision Vostro
+gid: 2
+*************************** 2. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Elitebook Probook
+gid: 1
+––– input –––
+export INSTANCE=5
+––– output –––
+––– block: ../base/flexible/stop-searchd –––
+––– input –––
+mysql -h0 -P1306 -e "INSERT INTO replication:testrt VALUES(0,'List of HP business laptops','Macbook',3);"; echo $?
+––– output –––
+0
+––– input –––
+export INSTANCE=5
+––– output –––
+––– block: ../base/flexible/start-searchd –––
+––– input –––
+sleep 5; mysql -h0 -P5306 -e "SELECT * FROM testrt\G;"
+––– output –––
+*************************** 1. row ***************************
+id: %{NUMBER}
+title: List of Dell business laptops
+content: Latitude Precision Vostro
+gid: 2
+*************************** 2. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Macbook
+gid: 3
+*************************** 3. row ***************************
+id: %{NUMBER}
+title: List of HP business laptops
+content: Elitebook Probook
+gid: 1
+––– input –––
+export INSTANCE=2
+––– output –––
+––– block: ../base/flexible/stop-searchd –––
+––– input –––
+mysql -h0 -P3306 -e "INSERT INTO replication:testrt VALUES(0,'NEW','TABLE',4);"; echo $?
+––– output –––
+0
+––– input –––
+export INSTANCE=2
```

Comment:
INSTANCE already exported in line 98, you don't need to redefine this variable again